### ISSUE TITLE
Write docs about private installations

### ISSUE THREAD

**Issue by teemow** [giantswarm/roadmap#1695]
We need to explain how our private cluster setup works and put this into our docs.

**Comment by teemow** [comment]
Related issue (handout for kubecon): https://github.com/giantswarm/giantswarm/issues/26101

**Comment by teemow** [comment]
Also related: https://github.com/giantswarm/giantswarm/issues/24847

**Comment by pipo02mix** [comment]
@alex-dabija do we write something? Or we leave till we have a new lead?

**Comment by alex-dabija** [comment]
> [@alex-dabija](https://github.com/alex-dabija) do we write something? Or we leave till we have a new lead?

I think documentation is needed because if I'm not mistaken we are doing something with private clusters for one of our customers Tobiasz knows better the current state. @T-Kukawka what do you think?

**Comment by T-Kukawka** [comment]
yeah, we do have one customer installation on AWS, plus one on Azure that is private 😅

**Comment by pipo02mix** [comment]
ok I will write a draft and ask phoenix for completion

**Comment by pipo02mix** [comment]
I moved this to the block till we don't really have an interest from marketing or a lead to write about private installations
